src/NzbDrone.Core/Music/TrackRepository.cs

```diff
@@ -16,6 +16,7 @@ public interface ITrackRepository : IBasicRepository<Track>
         Track Find(int artistId, int albumId, int trackNumber);
         List<Track> GetTracks(int artistId);
         List<Track> GetTracks(int artistId, int albumId);
+        Track GetTrackByArtist(int artistId, int trackNumber);
```

Comment:
Can't reliably get track based on just artistid and track number. Ex. Alien Ant Farm track 1 could be track 1 on any of their albums. We need album title or Id as well.